src/rust/inetstack/protocols/layer4/tcp/established/sender.rs

```diff
@@ -393,68 +402,96 @@ impl Sender {
         cb.emit(header, Some(data));
     }
 
-    // This segment acknowledges new data, so process the ack.
-    pub fn process_ack(&mut self, mut cb: SharedControlBlock, header: &TcpHeader, now: Instant) {
-        let bytes_acknowledged: u32 = (header.ack_num - self.send_unacked.get()).into();
-        let mut bytes_remaining: usize = bytes_acknowledged as usize;
-        // Remove bytes from the unacked queue.
-        while bytes_remaining != 0 {
-            if let Some(mut segment) = self.unacked_queue.try_pop() {
-                // Add sample for RTO if we have an initial transmit time.
-                // Note that in the case of repacketization, an ack for the first byte is enough for the time sample.
-                // TODO: TCP timestamp support.
-                if let Some(initial_tx) = segment.initial_tx {
-                    cb.rto_add_sample(now - initial_tx);
-                }
+    // Process an ack.
+    pub fn process_ack(&mut self, header: &TcpHeader, now: Instant) {
+        // Process the ACK.
```

Comment:
These comments: ("Process the ack") can be removed because they don't add any context.